### ISSUE TITLE
Provide a way to skip gcov execution

### ISSUE THREAD

**Issue by lpenz** [codecov/codecov-action#73]
There's a very high coupling between the version of gcov and the version of gcc used during compilation. If they don't match, we get the following error:

    ./CMakeCXXCompilerId.gcno:version 'A83*', prefer 'A93*'

We have to run gcov  in the same container that compiled the tests.
Codecov's bash starter runs gcov by default, but it has a flag that, when set, skips that phase.
Unfortunately, the github codecov-action doesn't have a similar flag as far as I could tell.

TL;DR: can we get something that allows us to tell codecov-action to not run gcov and just use the gcov files that are already there? That detaches us from having to use the same compiler present in codecov-action's environment.
